internal/db_impl/sqlite3/client.go

```diff
@@ -289,3 +289,22 @@ func pathExists(path string) (bool, error) {
 func getDatabaseConn(dir, userID, path string) string {
 	return fmt.Sprintf("file:%v?cache=shared&_fk=1&_journal=WAL", path)
 }
+
+func TestUpdateDBVersion(ctx context.Context, dbPath, userID string, version int) error {
```

Comment:
should this be in `client_test.go` ?

Reply:
The logic that verifies if the db is corrupted and erases it in `backend.go`. To check for this I have to start the server add the user and observer the behavior. I need access to a protected method inside the SQLite implementation.

Alternatively, I need to add a method to the `db` interface to write the database version, which I don't think is a good idea 👀